repository.yml

```diff
@@ -34,11 +34,11 @@ repo.newt_compatibility:
     0.0.0:
         0.0.0: good
     0.0.3:
-        1.7.0: good
```

Comment:
No, we don't change what past versions versions is compatible with, only master and future versions ;)

Reply:
Yes, mynewt-core 1.8.0 is only referenced in repository.yml when we tag a release in uwb-apps that is verified to work.